M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1440,76 +1519,100 @@ doc ///
             the rank of the submatrices considered.  See @TO [getSubmatrixOfRank, Strategy]@.
-            In the future, we hope to multithread this function, in which case the threading would be controlled by the option {\tt Threads}.
+            In the future, we hope to multithread this function, in which case the threading would be controlled by the option {tt Threads}.
     SeeAlso
         isRankAtLeast
         [getSubmatrixOfRank, Strategy]
 ///
 
 doc ///
     Key
-        regularInCodimension
-        (regularInCodimension, ZZ, Ring)
-        [regularInCodimension, Verbose]
-        [regularInCodimension, ModP]
-        [regularInCodimension, MinMinorsFunction]
-        [regularInCodimension, CodimCheckFunction]
-        [regularInCodimension, PairLimit]
-        [regularInCodimension, UseOnlyFastCodim]
-        [regularInCodimension, SPairsFunction]
+        PointOptions
+        [chooseGoodMinors, PointOptions]
+        [getSubmatrixOfRank, PointOptions]
+        [isRankAtLeast, PointOptions]
+        [projDim, PointOptions]
+        [Rn, PointOptions]
+    Headline
+        options to pass to functions in the package RandomRationalPoints
+    Description
+        Text
+            {\tt PointOptions} is an option in various functions in this package, which can store options to be passed to the function {\tt findANonZeroMinor} in {\tt RandomRationalPoints}.  
+        Example
+            (options Rn)#PointOptions
+            options findANonZeroMinor
+        Text
+            Notice the field is allowed to be extended by default.  Furthermore, we have set {\tt Homogeneous=>false} by default, and set {\tt ProjectionAttempts => 0}.  While generic linear  projection provides good median time in the examples we tried, in some cases it had extremely long run times.
+    SeeAlso
+        findANonZeroMinor
+///
+
+doc ///
+    Key
+        Rn
+        (Rn, ZZ, Ring)
+        [Rn, Verbose]
+        [Rn, ModP]
+        [Rn, MinMinorsFunction]
+        [Rn, CodimCheckFunction]
+        [Rn, PairLimit]
+        [Rn, UseOnlyFastCodim]
+        [Rn, SPairsFunction]
         MinMinorsFunction
         CodimCheckFunction
         UseOnlyFastCodim
     Headline
         attempts to show that the ring is regular in codimension n
     Usage
-        regularInCodimension(n, R)
+        Rn(n, R)
     Inputs
         n: ZZ
         R: Ring
     Outputs
         : Boolean
-            {\tt true}, if the ring is regular in codimension n ($R_n$), {\tt false} if it determines it is not, {\tt null} if no determination is made 
+            true, if the ring is Rn, false if it determines it is not
+        : null
+            if no determination is made
     Description
         Text
-            This function looks at interesting minors of the jacobian matrix to try to verify that the ring is $R_n$.
+            This function looks at interesting minors of the jacobian matrix to try to verify that the ring is Rn.
```

Comment:
Rn is not an English word.  Rephrase to "the ring satisfies the property $R_n$".

Reply:
I have just changed it to regular in codimension n.